autorag/evaluator.py

```diff
@@ -155,7 +156,18 @@ def evaluate(config, qa_data_path, corpus_data_path):
     logger.info('Evaluation complete.')
 
 
-cli.add_command(evaluate)
+@click.command()
+@click.option('--yaml_path', type=str, help='Path to extracted yaml file.')
```

Comment:
Of course, the only Yaml file format used in our library is the pipeline, but I would like you to mention that it is a variable about the pipeline in the description of the variable.

I think it's hard to read the code internally to make the variable name as Pipeline

Reply:
How about config_path?